templates/demo/app/dapp.js

```diff
@@ -44,10 +44,15 @@ class App extends React.Component {
           });
         }
       }
+
+      EmbarkJS.Storage.isAvailable().then((result) => {
```

Comment:
Would this be a good opportunity to only show IPNS functionality when IPFS is used as the storage provider?